packages/shift-react-components/src/components/account/sidebar-item.js

```diff
@@ -5,14 +5,14 @@ import classNames from 'classnames'
 import Config from '../../lib/config'
 import link from '../../objects/link'
```

Comment:
Don't need `link`  or `config` anymore I believe. If so you can get rid of line 11 too!